<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProtocolDialog</class>
 <widget class="QDialog" name="NewProtocolDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Protocol</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="topFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="bodyFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="nameHLayout">
           <item>
            <widget class="QLabel" name="protocolLabel">
             <property name="text">
              <string>Protocol Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="protocolLineEdit">
             <property name="toolTip">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Protocol1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="typeHLayout">
           <item>
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type                </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="typeLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="versionHLayout">
           <item>
            <widget class="QLabel" name="versionLabel">
             <property name="text">
              <string>Version            </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="versionLineEdit">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="descriptionHLayout">
           <item>
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Description     </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="descriptionLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="detailedHLayout">
           <item>
            <layout class="QVBoxLayout" name="detailedVLayout">
             <item>
              <widget class="QLabel" name="detailDescLabel">
               <property name="text">
                <string>Detailed Desc </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>48</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="detailDescTextEdit">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
             <property name="overwriteMode">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="buttonFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="buttonVLayout">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonHLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>178</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="createButton">
             <property name="text">
              <string>Create</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
